packages/application-extension/style/base.css

```diff
@@ -17,3 +17,17 @@
   border: none;
   align-items: center;
 }
+
+#jp-Overlay {
+  position: absolute;
+  top: 67%;
+  left: 45%;
+  width: 47%;
+  height: 32%;
+  border-radius: 25%;
+  z-index: inherit;
+  text-align: center;
+  color: rgb(255, 255, 255);
+  background: rgb(51, 51, 51);
```

Comment:
Can it use a variable from jupyterlab variables?

Reply:
I have refactored the code to use Css variables